website/project/model.py

```diff
@@ -557,6 +569,8 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
         ]
     }]
 
+    is_draft_registration = False
```

Comment:
Project transfer note for followup: ensure that I'm clear on the backend dataflow: differences and interplay between the various classes, eg `DraftRegistration` vs `Node.is_draft_registration`

Left off here for the evening. :ghost: